core/src/test/java/org/apache/calcite/test/ScannableTableTest.java

```diff
@@ -445,6 +469,20 @@ protected ConnectionPostProcessor newSchema(final String schemaName,
     };
   }
 
+  protected ConnectionPostProcessor newSchema(final String schemaName,
+                                                     final String tableName1, final Table table1,
+                                                     final String tableName2, final Table table2) {
+    return connection -> {
+      CalciteConnection con = connection.unwrap(CalciteConnection.class);
+      SchemaPlus rootSchema = con.getRootSchema();
+      SchemaPlus schema = rootSchema.add(schemaName, new AbstractSchema());
+      schema.add(tableName1, table1);
```

Comment:
This method signature is weird, can we make more effort to refactor it out? Also be cautious to the indentation

Reply:
How about a list of tableNames and a list of Tables?

Reply:
That's good idea.